geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/publishAndSubscribe/PubSub.java

```diff
@@ -96,7 +96,9 @@ long publish(
    */
   List<byte[]> findSubscriptionNames(Client client);
 
-  List<byte[]> findChannelNames();
+  List<Object> findChannelNames();
 
-  List<byte[]> findChannelNames(byte[] pattern);
+  List<Object> findChannelNames(byte[] pattern);
```

Comment:
These can (and should, I think) go back to `List<byte[]>` if the change in `PubSubExecutor` is applied.